workers/unity/Packages/com.improbable.gdk.core/Systems/WorkerSystem.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Encapsulates a SpatialOS worker instance.
```

Comment:
I don't think you need to say it encapsulates it. That is assumed it be being a class that has code and stuff.
Also I really want to make this system add the other core systems.

Reply:
Also noticed a missed a few docstrings here